src/registrar/templates/includes/form_errors.html

```diff
@@ -1,5 +1,9 @@
+{% comment %} 
```

Comment:
**question**: @igorkorenfeld Can you confirm that we don't want the individual field errors to appear at the top of the form page any longer? I think that we have heard from some of our user interviews that people find it helpful to know at the top of the page what they need to fix. As it is, this PR looks like:

<img width="684" alt="Screen Shot 2023-02-17 at 12 11 31" src="https://user-images.githubusercontent.com/443389/219744436-49b09331-c8b7-48f6-a37c-f5d9d2592d15.png">

Reply:
I was hoping to have two behaviors:
1. On "Save progress" - show warning,  but don't list out the errors.
2. On "Save and continue" - show error and list out the errors. 

I think "Save progress" should has a softer warnings because the user isn't trying to advance yet. But on "Save and continue" we do want to be more detailed so that the user can fix the errors before they advance. 

Does that make sense?

Reply:
This will require new ticket. It requires more logical in coding.